test/example.test.js

```diff
@@ -6,13 +6,14 @@ const test = QUnit.test;
 test('time to test a function', (expect) => {
     //Arrange
     // Set up your arguments and expectations
-    const expected = true;
+    const guess = true;
     
     //Act 
     // Call the function you're testing and set the result to a const
-    const actual = false;
-
-    //Expect
+    const correctNumber = false;
```

Comment:
This is great logic but you don't need to write a function like this in a test! Basically all you need to do is import your functions from your utils files, call them, feed them some fake data, and see if they do their job!